src/NeoSharp.Application/Attributes/PromptCommandAttribute.cs

```diff
@@ -136,48 +136,66 @@ internal void SetMethod(MethodInfo method)
         /// Convert string arguments to Method arguments
         /// </summary>
         /// <param name="args">Arguments</param>
+        /// <param name="injector">Injector</param>
         /// <returns>Return parsed arguments</returns>
-        public object[] ConvertToArguments(CommandToken[] args)
+        public object[] ConvertToArguments(CommandToken[] args, IContainer injector)
```

Comment:
I think we can find a better way to do this. IContainer should never been injected.